### DISCUSSION TITLE
Elixir Node Server Porperties issue

### DISCUSSION THREAD

**Discussion by thebyteslayer** [pterodactyl/panel#4911] [Support] [answered]
The server changes any start the ip 0.0.0.0 and i cant connect to the server with my ip only with a tcp tunnel

**Comment by Sharktheone** [comment, marked as answer]
So I guess, you mean that any start IP changes to `0.0.0.0`. This should not matter for your use case, since what `0.0.0.0` does is it allows all IPs to connect to the Server. The reason why you can't connect to your server is probably because you are not in the same network and so need to allow connections to this port in your firewall (port forwarding).
